keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -244,25 +309,55 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
   Once CRI implementation provide the relevant RPC calls
   the e2e tests will not fail but need to be extended.
 
+- Once the initial Alpha release  CRI-O supports the
+  `CheckpointContainer` CRI RPC and tests have been
+  enhanced to support CRI implementation that implement
+  the `CheckpointContainer` CRI RPC
+
+- Once Kubernetes was released with the `CheckpointContainer` CRI RPC
+  CRI-O has been updated to support the new CRI RPC.
+  The tests have been enhanced to work with CRI implementations
+  that support the `CheckpointContainer` CRI RPC as well as
+  CRI implementations that do not support it. The tests also handle
+  if the corresponding feature gate is disabled or enabled:
+  <https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/checkpoint_container.go>
+
 ### Graduation Criteria
 
 #### Alpha
 
-- [ ] Implement the new feature gate and kubelet implementation
-- [ ] Ensure proper tests are in place
-- [ ] Update documentation to make the feature visible
+- [X] Implement the new feature gate and kubelet implementation
+- [X] Ensure proper tests are in place
+- [X] Update documentation to make the feature visible
+  - <https://kubernetes.io/docs/reference/node/kubelet-checkpoint-api/>
+  - <https://kubernetes.io/blog/2022/12/05/forensic-container-checkpointing-alpha/>
+  - <https://kubernetes.io/blog/2023/03/10/forensic-container-analysis/>
 
 #### Alpha to Beta Graduation
 
-At least one container engine has to have implemented the
-corresponding CRI APIs to introduce e2e test for checkpointing.
+At least one container engine implemented the corresponding CRI APIs:
+
+- [x] CRI-O
 
+In Kubernetes:
+
+- [x] No major bugs reported in the previous cycle
 - [ ] Enable the feature per default
-- [ ] No major bugs reported in the previous cycle
+- [ ] Add necessary metrics as described in the PRR sections
+  - [ ] Add CRI metrics
+  - [ ] Add kubelet metrics
 
```

Comment:
Can we add a section about the two tests you plan to add?

https://github.com/kubernetes/enhancements/pull/4288/files#r1480520984
https://github.com/kubernetes/enhancements/pull/4288/files#r1480543216